src/dbos-executor.ts

```diff
@@ -852,6 +853,25 @@ export class DBOSExecutor implements DBOSExecutorContext {
       }
     }
 
+    const handleWorkflowError = async (err: Error) => {
```

Comment:
We should be consistent here using either named functions (like `callPromiseWithTimeout` above) or arrow functions (like `runWorkflow` below). We can make `callPromiseWithTimeout` an arrow function in this PR or a separate PR.